get_all_manifests.sh

```diff
@@ -16,8 +16,8 @@ declare -A COMPONENT_MANIFESTS=(
     ["notebooks"]="opendatahub-io:notebooks:main:manifests:notebooks"
     ["trustyai"]="trustyai-explainability:trustyai-service-operator:main:config:trustyai-service-operator"
     ["model-mesh"]="opendatahub-io:modelmesh-serving:release-0.12.0-rc0:config:model-mesh"
-    ["odh-model-controller"]="opendatahub-io:odh-model-controller:release-0.12.0:config:odh-model-controller"
-    ["kserve"]="opendatahub-io:kserve:release-v0.12.1:config:kserve"
+    ["odh-model-controller"]="opendatahub-io:odh-model-controller:incubating:config:odh-model-controller"
+    ["kserve"]="opendatahub-io:kserve:release-v0.14:config:kserve"
```

Comment:
guess this is to track the related changes to include NIM right ?

Reply:
yes, the old branch i dont think has the change of nim-state included. and this is the change we have currently in "incubation branch"